docs/source/Howtos/Turn-based-Combat-System.md

```diff
@@ -389,7 +389,7 @@ def resolve_combat(combat_handler, actiondict):
     for (char, fleevalue) in flee.items():
         if fleevalue == 2:
             combat_handler.msg_all(f"{char} withdraws from combat.")
-            combat_handler.remove_character(char)
+            combat_handler.remove_character_from_playable_list(char)
```

Comment:
Why is this change needed on the `combat_handler`? That's a Script, not an account? Besides the combathandler's `remove_character` was not updated either. So this looks like an search-and-replace gone wrong.